week-2/square/square.go

```diff
@@ -0,0 +1,27 @@
+package square
+
+type Point struct {
+	x, y int
+}
+
+type Square struct {
+	start Point
+	a     uint
+}
+
+func New(x int, y int, a uint) Square {
```

Comment:
Why do you need this? You can create a new Square just by writing `s := Square{Point{1, 1}, 5}`

Reply:
because i creating this variable outside of file and i cannot setting inner variable `start` and `a` though `Square{Point{1, 1}, 5}` - it cause error. So i am creating constructor for this purpose